src/apiClient/index.ts

```diff
@@ -16,7 +16,7 @@ const parseParams = (params: object) => {
 
   keys.forEach(key => {
     const isParamTypeObject = typeof params[key] === 'object'
-    const isParamTypeArray = isParamTypeObject && params[key].length >= 0
+    const isParamTypeArray = isParamTypeObject && params[key] && params[key].length >= 0
```

Comment:
```suggestion
    const isParamTypeArray = isParamTypeObject && params[key]
```

Reply:
It should be enough to do a check like this.

Reply:
`Array.isArray(params[key])`?